undertow/src/main/java/org/wildfly/extension/undertow/ServletContainerDefinition.java

```diff
@@ -244,6 +244,7 @@ public Collection<AttributeDefinition> getAttributes() {
     public List<? extends PersistentResourceDefinition> getChildren() {
         return List.of(
                 new JspDefinition(),
+                new AffinityCookieDefinition(),
```

Comment:
This requires a model transformation and test case.

Reply:
Fixed, thanks.